packages/litexa-html/README.md

```diff
@@ -45,58 +41,147 @@ project_dir
+location of the start directive.
 
-This extension automatically adds the `ALEXA_PRESENTATION_HTML` interface declaration to
-your skill manifest upon deployment.
+For more information on what these values do, see [the directive documentation](https://developer.amazon.com/en-US/docs/alexa/web-api-for-games/alexa-presentation-html-interface.html#start-directive) 
 
-### Usage
 
-#### HTML.isHTMLPresent()
 
-This extension adds a global `HTML` object you can use in both Litexa and code files. It
-has one function, which is to detect whether or not HTML is supported on the user's device
-that the skill is running on. This is best used in conjunction with conditionally sending HTML
-directives, since you may want to fall back to APL if HTML is not supported on a device.
+
+### `HTML.isHTMLPresent()`
+
+`HTML.isHTMLPresent` will return true at runtime if the skill is currently running on 
+a device that supports the Alexa Web API. You can use this to write conditional 
+code that branches on whether the user will see your HTML.
 
 ```coffeescript
 launch
   if HTML.isHTMLPresent()
-    directive htmlStartDirective(webAppUrl) # returns `Alexa.Presentation.HTML.Start` directive; this will launch the web app
+    HTML "app.html"
+    -> startApp
   else
-    apl apl/splashScreen.json # the apl statement conditionally sends the directive based on if the device supports it
-  ...
+    say "Ah, I see your device doesn't support HTML. Let's try this instead."
+    -> alternativeExperience
 ```
 
-**WARNING**: Do not use APL and HTML directives in the same response. The user's device
-will likely experience undesired behavior.
+If at any time you need to shortcircuit HTML, you can call `HTML.setEnabled(false)`
+to make `isHTMLPresent()` subsequently report false regardless of the device capabilities. 
 
-#### Directives
+Note that by default Litexa test runs as if it were running
+on an Echo Show-like device, and will behave as if HTML is present. 
+You can specify an Echo Dot like device instead by calling `litexa test --device=dot`
 
-You can use the [`directive`](https://litexa.com/reference/#directive) keyword to add
-HTML directives to your skill responses.
+You can conditionally disable HTML in your Litexa tests by using the `DisableHTML` 
+test statement. This lets you create a single test suite that tests both conditions,
+as long as you test with a device that initially supports HTML.
 
-If you're interested in trying out transformers in your directives,
-but want to maintain a skill that can switch between an HTML device
-and a speaker only device, try using the usual Litexa `say` statements,
-then write a [post processor](http://litexa.com/reference/backdoor.html#modifying-the-alexa-response-object)
-to conditionally move your outputSpeech into a directive instead.
+```coffeescript
+TEST "with HTML"
+  DisableHTML false
+  launch
+
+TEST "without HTML"
+  DisableHTML true
+  launch
+```
```

Comment:
Intentionally not having an enable because I don't actually want to override the device choice, i.e. I don't want a testing scenario where you can do "litexa test --device=dot" but have HTML enabled. The option here is just to disable HTML, despite the device reporting HTML is available.

Reply:
Ah, that makes sense. I guess `disable false` just seems weird to me: Could perhaps go with `BlockHTML`/UnblockHTML` as an alternative, but fine with leaving it as is if you prefer.